src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -93,9 +96,28 @@ protected override bool Visit(ITestSkipped testSkipped)
 
         protected override bool Visit(ITestStarting testStarting)
         {
+            if (showProgress)
+            {
+                lock (consoleLock)
+                {
+                    Console.Error.WriteLine("   {0} [STARTING]", Escape(testStarting.Test.DisplayName));
+                }
```

Comment:
I didn't realize these things were getting written to stderr that seems wrong to me. Any ideas why they should go there as opposed to stdout?